src/ContainerInterface.php

```diff
@@ -33,5 +33,5 @@ public function get($id);
      *
      * @return bool
      */
-    public function has($id);
+    public function has(string $id);
```

Comment:
Would it be possible to add `:bool` as return type? Example:

```php
public function has(string $id): bool;
```

Reply:
This question is addressed in the last paragraph :)

Reply:
Ah thanks :-) I think this is handled on the basis of the new PHP version 7.2 but ok.